locales/de.json

```diff
@@ -165,9 +165,9 @@
     "sidebar.preferences": "Einstellungen",
     "sidebar.view_community_profile": "Community Profil anzeigen",
     "status.available": "Eine neue Version von itch ist verfügbar!",
-    "status.checking": "Es wird nach Aktualisierungen gesucht…",
-    "status.downloaded": "Klicke hier für Neustart und Update-Installation!",
-    "status.downloading": "Aktualisierung wird heruntergeladen…",
+    "status.checking": "Es wird nach Updates gesucht…",
+    "status.downloaded": "Klicke hier, um das Update anzuwenden,
```

Comment:
@fasterthanlime whoops! I forgot to add a `"` character!

Reply:
ha, that's why I wanted to finish https://github.com/itchio/itch/issues/2250

the good news is this repo doesn't trigger auto updates anywhere, so nothing broke